assets/template.pug

```diff
@@ -136,3 +136,28 @@ body
           span.school #{level.institution}
           | &nbsp;
           span.time #{level.startDate.split('-')[0]}&hairsp;&ndash;&hairsp;#{level.endDate.split('-')[0]}
+
+  if publications && publications.length
+    section.category
+      - var talksObj = publications.reduce((obj, publication) => {
+      -   obj[publication.name] = (obj[publication.name] || []).concat(publication)
+      -   return obj
+      - }, {})
```

Comment:
I copied the logic above on [L107-L110](https://github.com/testdouble/cobbler/blob/conference-talks/assets/template.pug#L107-L110) as an example to avoid setting any new patterns.

I think a good improvement would be to extract the transformation logic. That is, have one file where we take the data from CSON and transform it to the format expected by the template